minishift-ci-index.yaml

```diff
@@ -205,6 +205,10 @@
             name: RELEASE_VERSION
             default: ""
             description: "Release version to be released (Eg. 1.0.0)"
+        - bool:
+            name: SKIP_INTEGRATION
```

Comment:
@praveenkumar How it is going to use in minishift ? Could you light on the connection here?